common/src/main/scala/skinny/ParamType.scala

```diff
@@ -79,22 +79,24 @@ object ParamType {
     case v: Short => scala.math.BigDecimal(v)
   })
   case object String extends AbstractParamType({
+    case v: String if v.isEmpty => null // don't trim
```

Comment:
Yes, it does. I think this is reasonable enough in real world web apps. But we may miss some use cases.